easybuild/easyconfigs/s/Stampy/Stampy-1.0.28-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'Stampy'
+version = '1.0.28'
+
+homepage = 'http://www.well.ox.ac.uk/stampy'
+description = """Stampy is a package for the mapping of short reads from illumina sequencing machines onto
+ a reference genome."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+# download from http://www.well.ox.ac.uk/bioinformatics/Software/Stampy-latest.tgz and rename to Stampy-1.0.28.tgz
```

Comment:
this should be fixed:

http://www.well.ox.ac.uk/bioinformatics/Software/stampy-1.0.28r3475.tgz